src/main/java/com/salesforce/dev/pages/Campaigns/CampaignView.java

```diff
@@ -0,0 +1,88 @@
+package com.salesforce.dev.pages.Campaigns;
+
+import com.salesforce.dev.framework.DriverManager;
+import com.salesforce.dev.pages.Base.ViewBase;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebDriverException;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Created by veronica on 8/20/2015.
+ */
+public class CampaignView extends ViewBase {
+
+    public CampaignView(WebDriver driver) {
+        super.driver = driver;
+        super.wait = DriverManager.getInstance().getWait();
+        PageFactory.initElements(driver, this);
+        try {
+            wait.withTimeout(10, TimeUnit.SECONDS)
+                    .until(ExpectedConditions.visibilityOf(saveBtn));
+        } catch (WebDriverException e) {
+            throw new WebDriverException(e);
+        } finally {
+            wait.withTimeout(15, TimeUnit.SECONDS);
```

Comment:
http://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad